pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractReplicator.java

```diff
@@ -224,6 +224,10 @@ public static String getReplicatorName(String replicatorPrefix, String cluster)
         return (replicatorPrefix + "." + cluster).intern();
     }
 
+    public static String getClusterName(String replicatorPrefix, String replicatorName) {
+        return replicatorName.replace(replicatorPrefix + ".", "");
```

Comment:
Thisbis not valid.
We can strip out this string only if it is in the beginning

Reply:
fixed.